src/build_helper/src/fs/mod.rs

```diff
@@ -45,11 +51,34 @@ pub fn recursive_remove<P: AsRef<Path>>(path: P) -> io::Result<()> {
     #[cfg(not(windows))]
     let is_dir_like = fs::Metadata::is_dir;
 
-    if is_dir_like(&metadata) {
-        fs::remove_dir_all(path)
-    } else {
-        try_remove_op_set_perms(fs::remove_file, path, metadata)
+    const MAX_RETRIES: usize = 5;
+    const RETRY_DELAY_MS: u64 = 10;
```

Comment:
Suggestion: I'd bump it to 100ms even, I expect the retry to fire on this *very* infrequently.